app/main.py

```diff
@@ -2,0 +1,17 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: [dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
This line will return immediately if any friend is not vaccinated, and the function will not check the remaining friends. If you want to check all friends and then return a message, you should modify your logic. For example, you can add a flag to check if all friends are vaccinated.